src/api/app/controllers/webui/apidocs_controller.rb

```diff
@@ -20,6 +20,6 @@ def file
       flash[:error] = "File not found: #{params[:filename]}"
       redirect_back(fallback_location: {controller: 'apidocs', action: 'index'})
     end
-    return
+    nil
```

Comment:
Do you want to keep the return? :confused:

That is what the [ruby style guide ](https://github.com/bbatsov/ruby-style-guide#no-explicit-return)says: 

> Avoid return where not required for flow of control

And there are not configuration options for this cop, so the only alternative is not to enable the cop.